data/moves.ts

```diff
@@ -3283,6 +3285,7 @@ export const BattleMovedex: {[moveid: string]: MoveData} = {
 		},
 		secondary: null,
 		target: "normal",
+		selfDamage: "50% (if Ghost)",
```

Comment:
you originally had this called "half" / "50%", but that seems a little misleading because its only when Ghost so I expanded the desc

Reply:
oh, I would just remove it completely, just set `move.selfDamage = "50%"` if the user is a Ghost.

Reply:
then its not reflected in the data, only an ActiveMove thing

Reply:
It's not data in the first place, it's implementation detail...

Reply:
If this is for the benefit of search engines, you could set it to `"50%"` and remove it if the user isn't a Ghost, instead.